applications/nrf_desktop/description.rst

```diff
@@ -0,0 +1,1321 @@
+
+.. tabs::
+
+   .. tab:: Gaming mouse
+
+      .. figure:: /images/nrf_desktop_gaming_mouse_debug_board_slot.svg
+         :alt: nRF Desktop gaming mouse (top view)
+
+      The debug slot is located at the end of the gaming mouse, below the cover.
+
+   .. tab:: Desktop mouse
+
+      .. figure:: /images/nrf_desktop_desktop_mouse_side_debug.svg
+         :alt: nRF Desktop desktop mouse (side view)
+
+      The debug slot is located on the side of the desktop mouse.
+      It is accesible through a hole in the casing.
+
+   .. tab:: Keyboard
+
+      .. figure:: /images/nrf_desktop_keyboard_back_debug.svg
+         :alt: nRF Desktop keyboard (back view)
+
+      The debug slot is located on the back of the keyboard.
+
+..
+
+The boards that you can plug into these slots are shown below.
+You can use the debug board for programming the device (and powering it).
+The bypass boards are needed to make the device work when the debug board is not used.
+Their purpose is to close the circuits, which allows the device to be powered, for example during :ref:`nrf_desktop_testing_steps`.
+
+.. tabs::
+
+   .. tab:: Debug board
+
+      .. figure:: /images/nrf_desktop_400391_jlink_debug.svg
+         :alt: nRF Desktop debug board
+
+      The device can be programmed using the J-Link.
+      The J-Link connector slot is located on the top of the debug board.
+
+   .. tab:: Short bypass board
+
+      .. figure:: /images/nrf_desktop_400398_debug.svg
+         :alt: nRF Desktop bypass board (short)
+
+      The shorter nRF desktop bypass board can be used with the desktop mouse.
+
+   .. tab:: Long bypass board
+
+      .. figure:: /images/nrf_desktop_400398_long_debug.svg
+         :alt: nRF Desktop bypass board (long)
+
+      The longer nRF desktop bypass board can be used with the gaming mouse.
+
+..
+
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+The nRF Desktop peripherals are either suspended or powered off when they are not in use for a defined amount of time:
+
+* In the suspended state, the device maintains the active connection.
+* In the powered off state, the CPU is switched to the off mode.
+
+In both cases, most of the functionalities are disabled.
+For example, LEDs are turned off and advertising is stopped.
+
+Moving the mouse or pressing any button wakes up the device and turns on the disabled functionalities.
+
+You can define the amount of time after which the peripherals are suspended or powered off using the :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` Kconfig option.
+By default, this period is set to 120 seconds.
+
+.. important::
+    When the gaming mouse is powered from USB, the power down timeout functionality is disabled.
+
+    If a nRF Desktop device supports remote wakeup, the USB connected device goes to suspended state when USB is suspended.
+    The device can then trigger remote wakeup of the connected host on user input.
+
+Configuration
```

Comment:
> Maybe we could move `Application-specific Kconfig configuration` section content to a separate page and link there from here? So that this page would provide only high-level overview for getting started.

We could move this section to the "Application Kconfig options scheme" page since this page displays all of the application specific Kconfig?

Reply:
I wonder if it would not be better to move it to a separate page (this section is a high-level overview of "global" nRF Desktop configuration options, it does not focus on application module specific options). The mentioned existing page is a complete list of available application-specific Kconfig options (including all of the Kconfigs that belong to application-specific modules).

Reply:
Thanks Marek. The structure looks good now.